src/azure-openapi-linter/rules/ArraySchemaMustHaveItems.ts

```diff
@@ -12,19 +12,14 @@ rules.push({
   name: ArraySchemaMustHaveItems,
   severity: "warning",
   category: "SDKViolation",
-  mergeState: MergeStates.individual,
+  mergeState: MergeStates.composed,
   openapiType: OpenApiTypes.arm,
-  appliesTo_JsonQuery: "$.definitions.*.properties.*.type",
+  appliesTo_JsonQuery: "$.definitions.*.properties[?(@.type)]",
```

Comment:
The info is provided in readme docs, a simple tutorial is available [here](https://www.npmjs.com/package/jsonpath)